CRM/Eventsextras/Form/Settings.php

```diff
@@ -1,22 +1,21 @@
 <?php
```

Comment:
Changing the namespace to `CRM/EventsExtras` will also require renaming the `Eventextras` Folder to `EventExtras`, For consistency, Also there might be issues loading the classes if we don't do this on some other OS.

Reply:
It should be EventExtras already.